### PR TITLE
Add config option to specify message properties

### PR DESCRIPTION
**Pull request by jseely**

### CODE REVIEW

**Comment by toddysm** [comment]
@jseely Can you please add documentation how this config option is supposed to be used?

**Comment by jseely** [comment]
@toddysm updated. Let me know if it needs any more detail.

File: lib/fluent/plugin/azureeventhubs/http.rb [hunk 1]
```diff
@@ -44,11 +44,18 @@ def generate_sas_token(uri)
...
+  end
+
+  def send_w_properties(payload, properties)
     token = generate_sas_token(@uri.to_s)
     headers = {
       'Content-Type' => 'application/atom+xml;type=entry;charset=utf-8',
       'Authorization' => token
     }
+    if not properties.nil?
+      headers = headers.merge(properties)
```
Comment (toddysm) [inline lib/fluent/plugin/azureeventhubs/http.rb:57]: @jseely Just to be clear - those are the properties that you have in mind:
https://msdn.microsoft.com/en-us/library/azure/dn790676.aspx
correct? Shouldn't they be part of the message body? I see you are merging those in the headers

**Comment by jseely** [comment]
@toddysm @htgc any additional changes needed to have this merged?

**Comment by toddysm** [comment]
I will pull this in at the end of this month's release

**Pull request merged**